### PR TITLE
[back] docs: the install doc invites explicitly the users to configure the settings

### PR DESCRIPTION
**Pull request by bbosson**
Fixed little errors in the readme when setup project

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -50,13 +50,16 @@ Django application; and how to install and configure a PostgreSQL server.
...
-  [example](backend/documentation/settings-tournesol.yaml) in documentation
+  [example](documentation/settings-tournesol.yaml) in documentation
   folder. You can put this file in different locations:
     - `/etc/tournesol/settings-tournesol.yaml` (linux),
     - Anywhere but add the `path/to/file` in SETTINGS_FILE environment
       variable,
     - Anywhere but add the `path/to/file` in SETTINGS_FILE variable `.env`
       file.
+  
+- Add in the file `settings-tournesol.yaml`, custom values for the variable `CORS_ALLOWED_ORIGINS`
```
Comment (tbrnd) [inline backend/README.md:61]: For my understanding, why was this needed?

Comment (bbosson) [reply backend/README.md:61]: This is a variable to be customized because in the example provided (documentation/settings-tournesol.yaml) this variable is empty. It is necessary to satisfy the CORS when calling Front to Back

Comment (GresilleSiffle) [reply backend/README.md:61]: That's right, the back end only accepts cross origin requests if their origins have been explicitly configured.

Thank you @bbosson for taking the time to improve the documentation. As they are other variables that should be configured like `MEDIA_ROOT`, `STATIC_ROOT`, etc. I think we could invite the reader to simply configure the settings as they are described in the `documentation/settings-tournesol.yaml`.

If you don't mind, I'm going to adapt the README now : )

**Review by GresilleSiffle — APPROVED** [review]
